content/azure-devops-dependencies.md

```diff
@@ -0,0 +1,152 @@
+
+> ![Backlog 1](../images/azure-devops-dependencies-3.png) 
+
+Looking at their team backlog there is no visual evidence of the Raptor v1 engine dependency.
+
+> ![Query](../images/azure-devops-dependencies-4.png) 
+
+The team uses a [WIQL Query](https://docs.microsoft.com/en-us/azure/devops/boards/queries/wiql-syntax?view=azure-devops) to track and manage their and other team dependencies.  
+
+> ![Query Edit](../images/azure-devops-dependencies-5.png) 
+
+The dependency query is using the **Work Items and direct links** query type and looks for **predecessor** link type.
+
+>
+> **QUERY-BASED DEPENDENCY TRACKING** 
+> 
+> - PROS
+>   - Their backlog and board only show what they, as team 1, are responsible to deliver.
+>   - There is no noise or **waste** on their Kanban board.
+>   - Their dependency query has a consistent look & feel as their backlog view.
+> 
+> - CONS
+>   - Their backlog and board show no visual cue of the Raptor v1 engine dependency. 
+>
+
+---
+
+# Delivery Team 2 - Using Dependency Work Items
+
+Delivery team 2 has opted to create a dependency work item (489410) and a Story Enabler (489417) work item for the Raptor v2 engine they need from the Build Engines team. The dependency work item (489410) is used as a placeholder work item on the team's backlog linked to the Story Enabler (489417) which is pulled by the Engines Common Services team to their backlog. They use the same placeholder+dependency pair process to add the Booster's dependency on Merlin engines.
+
+> ![Backlog 2](../images/azure-devops-dependencies-6.png) 
+
+Looking at team 2's backlog, we note the following:
+
+1. Dependency work item (489410) for v2 Raptor engine is on the backlog, linked as a child to 2-stage Feature (489407).
+2. Raptor v2 engine Story Enabler (489417) is gone - pulled to the Engine team's backlog.
+3. Dependency work item (489411) for Merlin engine is on the backlog, linked as a child to Booster Feature (489408).
+4. Merlin engine Story Enabler (489419) is still on the team 2 backlog, which implies that the common services Engine team is not able to commit to the work item yet.
+
+> ![Board 2](../images/azure-devops-dependencies-7.png) 
+
+Switching to the **Feature** board view, we have a visual cue on both the 2-stage and Booster cards that they have dependencies. Users can click on the dependencies to get more detail and to follow the dependency links.
+
+> ![Dependency Extension](../images/azure-devops-dependencies-10.png) 
+
+The team also uses the optional [Dependency Tracker](https://marketplace.visualstudio.com/items?itemName=ms-eswm.dependencytracker) extension to get visual information on their dependencies, timelines, and risks. The extension is not for the faint hearted but delivers phenomenal dependency details when used currently.
```

Comment:
"...delivers phenomenal dependency details when used currently." ... should that be ".... used correctly"?

Reply:
@MerlinDarkstar looking at the master and the live blog, this was corrected a while ago:
![image](https://user-images.githubusercontent.com/12176555/204588339-c7222b24-60ba-416c-b625-f233eb630317.png)